### PR TITLE
added image links on README file

### PR DESCRIPTION
**Pull request by braulio94**
Links for each image on `README.md` so people can access the source code easily.

### CODE REVIEW

**Comment by braulio94** [comment]
@googlebot `I signed it!`

File: README.md [hunk 1]
```diff
@@ -27,11 +27,21 @@ beautiful Material Design and
 Cupertino (iOS-flavor) widgets, rich motion APIs,
 smooth natural scrolling, and platform awareness.
 
+<a href="https://github.com/flutter/flutter/tree/master/examples/flutter_gallery/lib/demo/animation">
```
Comment (mit-mit) [inline README.md]: Can we use markdown formatting for these links, please?

File: README.md [hunk 2]
```diff
@@ -27,11 +27,21 @@ beautiful Material Design and
...
 <img src="https://flutter.io/images/homepage/screenshot-1.png" width="270" height="480" alt="Brand-first shopping design" align="left">
+</a>
+
+<a href="https://github.com/flutter/posse_gallery">
```
Comment (mit-mit) [inline README.md]: As above

File: README.md [hunk 3]
```diff
@@ -27,11 +27,21 @@ beautiful Material Design and
...
 <img src="https://flutter.io/images/homepage/screenshot-2.png" width="270" height="480" alt="Fitness app design">
+</a>
 
+<a href="https://github.com/flutter/flutter/blob/master/examples/flutter_gallery/lib/demo/contacts_demo.dart">
```
Comment (mit-mit) [inline README.md]: As above

File: README.md [hunk 4]
```diff
@@ -27,11 +27,21 @@ beautiful Material Design and
...
 <img src="https://flutter.io/images/homepage/screenshot-3.png" width="270" height="480" alt="Contact app design" align="left">
+</a>
+
+<a href="https://codelabs.developers.google.com/codelabs/flutter-firebase">
```
Comment (mit-mit) [inline README.md]: As above

**Review by mit-mit — CHANGES_REQUESTED** [review]
Thanks for contributing! Some smaller comments

File: README.md (hunk 1 above)
Comment (braulio94) [reply README.md]: @mit-mit  by using markdown formatting, we'll have to sacrifice image sizing

Comment (braulio94) [reply README.md]: `[alt](https://github.com/flutter/posse_gallery)`

meaning we can't add image size to markdown formatting

Comment (braulio94) [reply README.md]: @mit-mit  DONE SIR! 👍

**Review by braulio94 — COMMENTED** [review]
image links with markdown formatting

**Review by mit-mit — APPROVED** [review]

**Comment by mit-mit** [comment]
Thanks for the contribution @braulio94 !

**Comment by braulio94** [comment]
Happy to help!

**Pull request merged**